src/DeviceInterface.js

```diff
@@ -38,7 +41,31 @@ const createAttachTooltip = (getAutofillData, refreshAlias, addresses) => (form,
 let attempts = 0
 
 class InterfacePrototype {
+    // Data
```

Comment:
IMHO, I prefer using `jsdoc` for this class rather than unclear _single line_ commenting.